app/services/candidates/registrations/behaviours/placement_preference.rb

```diff
@@ -5,22 +5,36 @@ module PlacementPreference
         extend ActiveSupport::Concern
 
         included do
+          with_options unless: :dont_validate_availability? do
```

Comment:
This feels like an unneeded negative? couldn't this be `if: :validate_availability?`